.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -1,18 +1,32 @@
+---
 name: Report an issue
-description: Tell us about something that's not working the way we (probably) intend
+description:
+  Tell us about something that is not working the way we (probably) intend.
 body:
-  - type: input
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to Elixir.
+
+        Please, do not use this form for guidance, questions or support.
+        Try instead in [Elixir Forum](https://elixirforum.com),
+        the [IRC Chat](https://web.libera.chat/#elixir),
+        [Stack Overflow](https://stackoverflow.com/questions/tagged/elixir),
+        [Slack](https://elixir-slackin.herokuapp.com),
+        [Discord](https://discord.gg/elixir) or in other online communities.
+
+  - type: textarea
```

Comment:
Since `elixir --version` returns multiple lines, I converted this field into a textarea